packages/babel-plugin-transform-modules-amd/test/fixtures/amd/export-from-2/output.js

```diff
@@ -4,10 +4,13 @@ define(["exports", "foo"], function (_exports, _foo) {
   Object.defineProperty(_exports, "__esModule", {
     value: true
   });
-  Object.defineProperty(_exports, "foo", {
-    enumerable: true,
-    get: function () {
-      return _foo.foo;
-    }
-  });
+
+  function _export(key, get) {
+    Object.defineProperty(_exports, key, {
+      enumerable: true,
+      get
```

Comment:
Is is okay to use the shorthand syntax here, or does it have to be `get: get`?

Reply:
It's better to use ES5 syntax. We can potentially change it in Babel 8.

Reply:
Changed to ES5 `get: get` in 9d10002